src/datachain/fs/utils.py

```diff
@@ -0,0 +1,27 @@
+from typing import TYPE_CHECKING
+
+from fsspec.implementations.local import LocalFileSystem
+
+if TYPE_CHECKING:
+    from fsspec import AbstractFileSystem
+
+
+def isdir(fs: "AbstractFileSystem", path: str) -> bool:
```

Comment:
Renamed `isfile` to `isdir`. I think we are more interested to see if something is a directory or not than files.